src/ctia/http/routes/coa.clj

```diff
@@ -87,7 +111,9 @@
                 :path-params [id :- s/Str]
                 :header-params [{Authorization :- (s/maybe s/Str) nil}]
                 :capabilities :read-coa
-                (if-let [coa (read-store :coa (fn [s] (read-coa s id)))]
+                :identity identity
```

Comment:
I would prefer not to override a core fn name here. I would suggest `id-record`.

edit: I saw that `identity` is used in other routes. So that is not something to do in this PR. But I believe it would be nice to keep in mind we should make that change one day.

Reply:
I'm not going to fix coding style issues from previous feature PRs into this one, I keep that in mind but I won't make the change now.